src/flamenco/features/fd_features_generated.c

```diff
@@ -917,7 +917,8 @@ fd_feature_id_t const ids[] = {
   { .index      = offsetof(fd_features_t, vote_state_add_vote_latency)>>3,
     .id         = {"\x61\xd8\xd7\x1f\x81\x6f\x32\x74\x56\x37\x18\x3c\x8b\x7e\x6e\xaa\x28\x71\xb5\x22\x61\x11\xb0\x39\x6d\xb6\x4f\xad\x75\xad\x77\xc4"},
                   /* 7axKe5BTYBDD87ftzWbk5DfzWMGyRvqmWTduuo22Yaqy */
-    .name       = "vote_state_add_vote_latency" },
+    .name       = "vote_state_add_vote_latency",
+    .cleaned_up = 2000 },
```

Comment:
shall we change this to be the minor version? it's likely this will be 2.1

Reply:
we can, not sure it matters too much since its been activate for while anyways.